account_move_accrual_entry/tests/test_account.py

```diff
@@ -0,0 +1,264 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from . import common
+
+_logger = logging.getLogger(__name__)
+
+
+class TestInvoice(common.CommonAccountMultiVat):
+    # TEST CREATE ACCOUNT.MOVE
+    def test_create_account_move(self):
+        def create_acrrual_invoice(self):
+            invoice = self.env["account.move"].create(
+                {
+                    "journal_id": self.journal.id,
+                    "accrual_date": self.date,
+                    "move_type": "out_invoice",
+                    "invoice_line_ids": [
+                        (
+                            0,
+                            0,
+                            {
+                                "account_id": self.account_7000.id,
+                            },
+                        )
+                    ],
+                }
+            )
+
+            line = invoice.invoice_line_ids[0]
+            self.assertEqual(line.accrual_account_id, self.account_7000)
+            self.assertEqual(line.account_id, self.accrual_account_id)
```

Comment:
Only one assert is valid per test, please convert this on `assertList` or similar